tests/core/test_lightning_module.py

```diff
@@ -313,13 +313,33 @@ def __init__(self, spec):
 
 @RunIf(min_torch="1.10", skip_windows=True)
 def test_sharded_tensor_state_dict(tmpdir, single_process_pg):
+    # PROBLEM 1:
```

Comment:
cc @yifuwang as the original author in #8944

also cc @ananthsub in case you know who else could help here

Reply:
For the note, `PROBLEM 1` should be unrelated to the hang and be fixed in #11416.